scripts/generate_json_api.py

```diff
@@ -66,43 +66,144 @@ class Examples(CommandTestCase):
     async def play(self):
         r = self.recorder
 
+        # general sdk
+
+        await r(
+            'Get Status',
+            'status'
+        )
         await r(
-            'List your accounts.',
+            'Get Version',
+            'version'
+        )
+        await r(
+            'Get Settings',
+            'settings', 'get'
+        )
+
+        # accounts
+
+        await r(
+            'List your accounts',
             'account', 'list'
         )
 
         account = await r(
-            'Create an account.',
+            'Create an account',
             'account', 'create', '"generated account"'
         )
 
         await r(
-            'Remove an account.',
-            'account', 'remove', account['id']
+            'Add an account from seed',
+            'account', 'add', '"new account"', f"--seed=\"{account['seed']}\""
         )
 
         await r(
-            'Add an account from seed.',
-            'account', 'add', '"new account"', f"--seed=\"{account['seed']}\""
+            'Modify maximum number of times a change address can be reused',
+            'account', 'set', account['id'], '--change_max_uses=10'
         )
 
+        # addresses
+
         await r(
-            'Modify maximum number of times a change address can be reused.',
-            'account', 'set', account['id'], '--change_max_uses=10'
+            'List addresses in default account',
+            'address', 'list'
+        )
+
+        an_address = await r(
+            'Get an unused address',
+            'address', 'unused'
+        )
+
+        address_list_by_id = await r(
+            'List addresses in specified account',
+            'address', 'list', f"--account_id=\"{account['id']}\""
+        )
+
+        await r(
+            'Check if address is mine',
+            'address', 'is_mine', an_address
+        )
+
+        # send/fund
+
+        transfer = await r(
+            'Transfer LBC',
```

Comment:
Can you make this more descriptive: "Send 2 LBC from default account to a specific account."